pkgs/top-level/python-packages.nix

```
@@ -7434,6 +7434,8 @@ in {
 
   pyalmond = callPackage ../development/python-modules/pyalmond { };
 
+  py3amf = callPackage ../development/python-modules/py3amf { };
```

This allows you to take in inputs via `callPackage` instead of using `with python.pkgs;`

```suggestion
  py3amf = python3Packages.callPackage ../development/python-modules/py3amf { };
```